src/ts/layers/raster.ts

```diff
@@ -21,7 +21,7 @@ module powerbi.extensibility.visual {
                 source: 'raster',
                 type: 'raster'
             });
-            map.addLayer(rasterLayer, 'water');
+            map.addLayer(rasterLayer, beforeLayerId);
```

Comment:
@samgehret testing this out, where do dataviz layers get placed in relation to the WMS/WMTS layer?  Above this raster map layer?